pkgs/test_core/lib/src/runner/configuration/args.dart

```diff
@@ -27,13 +27,13 @@ final ArgParser _parser = (() {
   parser.addFlag('help',
       abbr: 'h', negatable: false, help: 'Shows this usage information.');
```

Comment:
Should this be `Show` for consistency?

```suggestion
      abbr: 'h', negatable: false, help: 'Show this usage information.');
```